packages/prisma/zod-utils.ts

```diff
@@ -361,6 +361,8 @@ export const teamMetadataSchema = z
     paymentId: z.string(),
     subscriptionId: z.string().nullable(),
     subscriptionItemId: z.string().nullable(),
+    orgSeats: z.number().nullable(),
+    orgPricePerSeat: z.number().nullable(),
```

Comment:
Should we move these to orgsettings table (I understand this was probably started before that was merged)

Reply:
This is temporarily required till user pays for the org. So, discussed that it is fine here.